### PR TITLE
Fix example and use actual sample files so that we are ready for doc …

### PR DESCRIPTION
**Pull request by esnible**
Resolves https://github.com/istio/istio.io/issues/8805

Fixes example by removing the `_`.  Example now uses actual sample files so that we are ready for [https://github.com/istio/istio.io/issues/8797](doc testing the page).

[x] User Experience

cc @therealmitchconnors I changed the title of this doc.  The original name was difficult for me to make sense of without reading the document.  Please take a look at this, especially the title change.

### CODE REVIEW

File: content/en/docs/ops/configuration/mesh/config-resource-ready/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Wait for Resource Status to Apply Configuration
-description: Describes how to wait to apply mesh configuration until a resource reaches a given status or readiness.
+title: Wait on Resource Status for Applied Configuration
+description: Describes how to wait until a resource reaches a given status or readiness.
```
Comment (frankbu) [inline content/en/docs/ops/configuration/mesh/config-resource-ready/index.md:3]:
```suggestion
description: Describes how to wait until a resource reaches a given status of readiness.
```

**Review by therealmitchconnors — APPROVED** [review]
+1, thank you, Ed!

File: content/en/docs/ops/configuration/mesh/config-resource-ready/index.md (hunk 1 above)
Comment (esnible) [reply content/en/docs/ops/configuration/mesh/config-resource-ready/index.md]: @frankbu I fixed this and did the `/retest` but something is failing unrelated to this change.  Thoughts?

**Review by frankbu — APPROVED** [review]
Failing tests unrelated to this PR.

**Pull request merged**